src/modules/shared/pages/innovation/sections/section-info.component.ts

```diff
@@ -18,21 +17,8 @@ import { getInnovationRecordConfig, innovationSectionsWithFiles } from '@modules
 })
 export class PageInnovationSectionInfoComponent extends CoreComponent implements OnInit {
 
-  innovation: ContextInnovationType;
-
-  section: {
-    id: InnovationSectionEnum,
-    nextSectionId: null | string,
-    title: string,
-    status: { id: keyof typeof INNOVATION_SECTION_STATUS, label: string },
-    isNotStarted: boolean,
-    submitButton: { show: boolean, label: string },
-    hasEvidences: boolean,
-    wizard: WizardEngineModel,
-    date: string,
-    submittedBy: null | { name: string, isOwner?: boolean },
-    openTasksCount: number
-  };
+  innovation: ContextInnovationType | undefined;
+  sectionId: string = '';
```

Comment:
sectionId doesn't have to be '' as you initialize it in the constructor

Reply:
makes sense!